build-in-container/Dockerfile

```diff
@@ -0,0 +1,41 @@
+FROM mcr.microsoft.com/onebranch/cbl-mariner/build:2.0
```

Comment:
Do we need to use onebranch? (can we even use this externally?)

Reply:
I'll replace this